packages/design-system/src/components/Dropdown/Dropdown.tsx

```diff
@@ -100,61 +75,77 @@ export interface DropdownProps {
-        );
-      }
+export const Dropdown: React.FC<DropdownProps> = (props: DropdownProps) => {
+  if (process.env.NODE_ENV !== 'production') {
+    // 'ariaLabel' is provided with a `label` prop that is not an empty string
+    if (props.ariaLabel && (typeof props.label !== 'string' || props.label.length > 0)) {
+      console.warn(
+        `Cannot use 'ariaLabel' and 'label' React properties together in the <Dropdown> component. If the 'label' prop is used, it should be written for all users so that an 'ariaLabel' is not needed. The 'ariaLabel' prop is intended to be used only when the input is missing an input label (i.e when an empty string is provided for the 'label' prop)`
+      );
+    }
+    // An empty string `label` is provided without a corresponding `ariaLabel` prop
+    if (!props.ariaLabel && typeof props.label === 'string' && props.label.length === 0) {
+      console.warn(
+        `Please provide an 'ariaLabel' when using the <Dropdown> component without a 'label' prop.`
+      );
+    }
+    if (props.children && props.options.length > 0) {
+      console.warn(
+        `Cannot use 'options' and 'children' React properties at the same time in the <Select> component. Please use 'children' for custom options and 'options' for general cases`
+      );
     }
   }
 
-  render() {
-    const containerProps: any = pick(this.props, FormControlPropKeys);
-    const inputOnlyProps: any = omit(this.props, FormControlPropKeys);
+  // Select specific props
+  const { ariaLabel, children, fieldClassName, options, size, ...selectProps } = props;
 
-    return (
-      <FormControl
-        {...containerProps}
-        component="div"
-        labelComponent="label"
-        render={({ id, errorId, setRef, errorMessage, errorPlacement }) => (
-          <Select
-            {...inputOnlyProps}
-            {...{ id, setRef, errorId, errorMessage, errorPlacement }}
-            inversed={this.props.inversed}
-          />
-        )}
-      />
-    );
-  }
-}
+  const optionElements =
+    children ??
+    options.map((option) => (
+      <option key={option.value} value={option.value}>
+        {option.label}
+      </option>
+    ));
+
+  const { labelProps, fieldProps, wrapperProps, bottomError } = useFormLabel({
+    ...selectProps,
+    labelComponent: 'label',
+    wrapperIsFieldset: true,
+  });
+
+  const selectClassNames = classNames(
+    'ds-c-field',
+    {
+      'ds-c-field--error': selectProps.errorMessage,
+      'ds-c-field--inverse': selectProps.inversed,
+    },
+    size && `ds-c-field--${size}`,
+    fieldClassName
+  );
+
+  const ref = useAutofocus<HTMLSelectElement>(props.autoFocus);
+
+  return (
+    <div {...wrapperProps}>
+      <FormLabel {...labelProps} fieldId={fieldProps.id} />
+      <select
+        aria-label={ariaLabel}
+        aria-describedby={fieldProps['aria-describedby']}
+        aria-invalid={fieldProps['aria-invalid']}
+        disabled={fieldProps.disabled}
```

Comment:
If we destructured `errorMessage` and `inversed` on line 103, would we be able to spread `fieldProps` directly onto the select? Right now the event listeners aren't getting applied besides it being more verbose.

Reply:
Related, the unit tests for Select will need to get migrated to Dropdown if we're removing Select

Reply:
Continued here: https://github.com/CMSgov/design-system/pull/2243#discussion_r1019681381